ci.sh

```diff
@@ -4,22 +4,37 @@
 
 set -euf -o pipefail
 
+spire_folder="spire-858d04b"
```

Comment:
Do you know `mktemp`? I think you could create a random directory with:
```bash
spire_folder=$(mktemp -d spire-XXXXXX)
```

(feel free to ignore, that's overengineering)

Reply:
that's not a random name, that's the name of the spire folder in the release tar, it was released with that name by error, it will be in a later version fixed to `spire-1.0.1`